test/functional/cmdLineTests/CDSAdaptorTest/build.xml

```diff
@@ -28,6 +28,9 @@
 		Build CDSAdaptorTest 
 	</description>
 	
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
```

Comment:
Does LIB have to be set before import getDependencies.xml?

Reply:
updated